### ISSUE TITLE
NameError in PoliciesAPI.template_details

### ISSUE THREAD

**Issue by j178** [tenable/pyTenable#136]
**Describe the bug**
`PoliciesAPI.template_details` in `tenable/io/policies.py` used an undefined variable `scan_id`, causing a NameError:
https://github.com/tenable/pyTenable/blob/master/tenable/io/policies.py#L68

**To Reproduce**
```py
from tenable.io import TenableIO

tio = TenableIO(access_key, secret_key)
tio.policies.template_details('advanced')
```

**Expected behavior**
Return the template details

**Screenshots**
![image](https://user-images.githubusercontent.com/10510431/61774284-2211e100-ae29-11e9-95aa-b369eb492247.png)

**System Information (please complete the following information):**
 - OS:  Ubuntu 
 - Architecture 64bit
 - Version 18.04
 - Memory  4G

**Additional context**

**Comment by SteveMcGrath** [comment]
good catch, the code was copied from the scan details method, however it apparently wasn't modified to properly handle policies.

**Comment by SteveMcGrath** [comment]
The fix will appear in 0.3.23.  If you can't wait, tell pip to pull from github ;)
